saleor/discount/tasks.py

```diff
@@ -214,3 +220,38 @@ def disconnect_voucher_codes_from_draft_orders_task(order_ids):
         ).filter(type=DiscountType.VOUCHER).delete()
         if remaining_ids := list(set(order_ids) - set(ids)):
             disconnect_voucher_codes_from_draft_orders_task.delay(remaining_ids)
+
+
+@app.task
```

Comment:
If you pass `name` here, you can keep it backwards-compatible.

Reply:
I kept the name unchanged.

Reply:
But Celery uses a full module path for automatic task names, doesn't it? So names are:
- old: `saleor.discount.migrations.tasks.saleor3_17.update_discounted_prices_task`
- new: `saleor.discount.tasks.update_discounted_prices_task`
So without explicitly setting the name, after this upgrade it will fail to resolve the task name (if the old tasks are still in the queue).